skyportal/utils/fink/poll_fink_alerts.py

```diff
@@ -0,0 +1,124 @@
+# coding: utf-8
+
+import sys
+
+import os
+
+from fink_client.consumer import AlertConsumer
+from fink_client.configuration import load_credentials
+
+from astropy.time import Time
+
+import post_fink_alerts
+
+token = sys.argv[1]
+
+
+def fid_to_filter(fid):
+    switcher = {1: 'ztfg', 2: 'ztfr', 3: 'ztfi'}
+    return switcher.get(fid)
+
```

Comment:
This is not universal - only works on ZTF data. Will need to make this accommodate other streams with other look up tables.